issues.py

```diff
@@ -97,12 +97,13 @@ def fetch_opened_pull_requests():
             r = requests.get(uri, auth = (config.api_token, 'x-oauth-basic'), headers = utils.caching_request_headers(uri))
         except requests.exceptions.RequestException:
             return []
-        if r.status_code == 304:
-            r = utils.cached_response(uri)
+        if r.status_code == 200:
+            utils.cache_response(r)
+        else:
+            r = utils.fetch_cached_response(uri)
```

Comment:
that's not correct. You need to put it to cache at 200, fetch from cache at 304 and show an error at everything else